packages/app/src/app/overmind/namespaces/dashboard/state.ts

```diff
@@ -65,8 +66,8 @@ export type State = {
     older: Sandbox[];
   };
   curatedAlbums: DashboardAlbum[];
-  // TODO: fix unknown
   contributions: DashboardContributionBranch[] | null;
+  v2Repositories: DashboardV2Repository[] | null;
```

Comment:
should we call them `repositories` to not carry the v2 thing everywhere?

Reply:
Removed `v2` references in commit [14c8fa7](https://github.com/codesandbox/codesandbox-client/pull/6917/commits/14c8fa7fe40935878d890de0e5008a6003b18ec7).